### ISSUE TITLE
Have a seperate badge for Patreon supporters that aren't supporting anymore

### ISSUE THREAD

**Issue by AshtonSparx** [Neos-Metaverse/NeosPublic#2005]
## Is your tweak request related to a problem? Please describe.
I've discovered that, users that aren't supporting Neos anymore, permanently keep the badge as if they're still supporting which can be misleading. Such as, they have the badge, but the Neos API marks them as no longer supporting.

## Describe what would you like tweaked
I think that users who supported Neos should still get a badge, but at the end of their term, it gets replaced with a new badge that shows they've supported before. Like a star or something as like, a thank you.

## Describe alternatives you've considered
Worst case scenario, to reduce confusion, remove the badge or at least, make default, un-modified name-plates not the Patreon colour.

## Additional context
N/A

**Comment by TehTurk** [comment]
It'd be neat, but I don't know the way it works right now would just be a good thank you for those who've supported previously while also not calling out folks for a situation they might not be okay with disclosing (sometimes people can't always support or financial issues)

**Comment by AshtonSparx** [comment]
Then that would go with the alternative, fully remove the badge and color change to avoid confusion

**Comment by TehTurk** [comment]
Yeah but that's the thing, by people being aware if someone not having money I see that being a problem socially. Plus we don't want badges having any extra meaning then they aready do.

**Comment by AshtonSparx** [comment]
I understand that but, it literally de-values the purpose of being Patreon if you're able to keep half of the benefits WHILE you're no longer a Patreon -- It also gives zero information what Patreon tier you were, could be a $1 tier for a month, who knows. It should literally have zero impact socially. If you wanted to tell someone what tier you were, that's up to the person. But if you stop supporting, you **_should not_** still be receiving basically half of your benefits if you choose to stop supporting. The better latter would be fully removing it, or replacing it with a different badge.

**Comment by Frooxius** [comment]
People who are no longer Patreons aren't receiving anywhere close to half the benefits of Patreon. There is no NCR mint, they have no extra storage, no access to Patreon stuff (channels, folders, builds) and so on.

It is only a badge to show they have supported the project - any support at any time helps, some people can do less, some can do more, some only for limited time, we appreciate all the support regardless.

That said, perhaps we can distinguish previous support from active one in some way visually, but exagerrating won't help the discussion about that.

**Comment by AshtonSparx** [comment]
> That said, perhaps we can distinguish previous support from active one in some way visually

That's all i was requesting <3 some sort of separate visual difference between a past Patreon and a current Patreon. I wasn't trying to exaggerate, I'm sorry if it came off as that
